README.md

```diff
@@ -2,39 +2,30 @@
 
 This guide will walk you through deploying a trivial
 [Dockerfile](https://docs.docker.com/reference/builder/) HTTP application
-on [Deis](https://github.com/deis/deis).
+on Deis Workflow.
```

Comment:
wouldn't it be good to have a link to the workflow project here.I saw you were adding them in additional resources.